reference/shaders-hlsl/vert/basic.vert

```diff
@@ -0,0 +1,47 @@
+struct _UBO
+{
+    float4x4 uMVP;
+};
+
+cbuffer UBO
+{
+    _UBO _16;
+};
+uniform float4 gl_HalfPixel;
```

Comment:
Aye and it needs the application to set gl_HalfPixel. It doesn't show up if shaders are compiled for D3D11.